cardano-cli/src/Cardano/CLI/EraBased/Run/Transaction.hs

```diff
@@ -420,8 +420,6 @@ runTxBuildRaw era
       <- first TxCmdTxFeeValidationError $ validateTxFee era mFee
     validatedLowerBound
       <- first TxCmdTxValidityLowerBoundValidationError (validateTxValidityLowerBound era mLowerBound)
-    validatedUpperBound
-      <- first TxCmdTxValidityUpperBoundValidationError (validateTxValidityUpperBound era mUpperBound)
```

Comment:
We no longer do validation here because by virtue of using era-sensitive types in the era-based CLI parser the values are already valid.

Reply:
Does this mean I should remove this line?  `validatedUpperBound <- ..` ?